docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '2.2'
+#version tag is obsolete
```

Comment:
Why not simply remove the line?

Reply:
Don't want people adding it back in, because they haven't gotten the memo yet.